containers/sandbox/Dockerfile

```diff
@@ -31,4 +32,4 @@ RUN ln -s /usr/bin/python3 /usr/bin/python
 
 # install basic dependencies for CodeActAgent
 RUN pip3 install --upgrade pip
-RUN pip3 install jupyterlab notebook jupyter_kernel_gateway flake8
+RUN pip3 install jupyterlab notebook jupyter_kernel_gateway flake8 python-docx PyPDF2 openpyxl python-pptx pylatexenc requests pandas openai opencv-python
```

Comment:
I actually think if there's a way for us to build two sandboxes, one can be a minimal one that is used for most agents that do not require these; another one can be tagged ":agent_skills" version, which includes this dependency.
I think installing a lot of these packages for `agentskills` on every sandbox initialization just take too much time - would be nice to have a specific version of container that already has these dependencies installed

Reply:
Yeah two sandboxes sounds like a plan

Reply:
```suggestion
RUN pip3 install jupyterlab notebook jupyter_kernel_gateway flake8
# TODO: those dependencies are needed for agentskills, we should pack them in a new sandbox image
RUN pip3 install python-docx PyPDF2 openpyxl python-pptx pylatexenc requests pandas openai opencv-python
```

Reply:
@Shimada666 could you see if we could achieve this easily? Thanks!!!

Reply:
If these are not installed in the Dockerfile in advance, installing the dependencies for agentskill will be relatively slow, probably 20+ seconds, affect the user experience. Of course, the image size has also increased significantly, 545M -> 1G...

In fact, I plan to merge agentskill with Jupyter later on, leaving only agentskill plugins. At that time, installing these dependencies will also be essential.

Alternatively, we can temporarily disable large packages, such as pandas and opencv, as discussed in the architecture below. These audio and video readers are only temporary solutions here; ultimately, interaction will rely on the server side. The sandbox will simply send a command to the REST server, and the specific invocation will be executed by the REST server.

Reply:
In the worst case, for simplicity, we can just not install these dependencies in the sandbox at all, since we have PIP cache mechanism.

But I'm kinda still leaning toward getting all those in at the moment and gradually refractor out these utilities to our action library (e.g., PDF reader), since the CodeActAgent at the main branch currently default to use `AgentSkills` so pre-installed dependencies could save some hassle.

Reply:
Yeah I can live with the status quo as long as we leave a comment for future reference

Reply:
Test Result

In the original state, installing dependencies takes 2 seconds
Excluding the opencv package, installing dependencies takes 8 seconds. If included in the image, the image size increases by 50M.
Including all the packages, installing dependencies takes 10 seconds. If included in the image, the image size increases by 168M.